content/en/apps/examples/anc.md

```diff
@@ -1,16 +1,55 @@
 ---
-title: "Antenatal Care Reference App"
-linkTitle: "ANC Reference App"
+title: "Antenatal Care"
+linkTitle: "Antenatal Care"
 weight: 1
 description: >
-  Reference application for CHWs using a mobile app for ANC
-keywords: anc
+  Antenatal care workflows for CHWs using a mobile app
```

Comment:
Having the term reference app in the description is useful.

Reply:
updated